CHANGELOG.md

```diff
@@ -20,6 +20,10 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 - Updates `Accordion` to close focused panel when "esc" key is pressed.
 - Updates `Accordion` to close panel when element within panel is focused and "esc" key is pressed.
 
+### Fixes
+
+- Fixes bug in `MultiSelect` where clicking a checkbox closes the component when `closeOnBlur` is true.
```

Comment:
I think this can be removed from the CHANGELOG.  See my response to your question in the component changelog.